todo/src/scripts/sortDirect.js

```diff
@@ -0,0 +1,28 @@
+export default class {
+    constructor(){
+        this.init();
+    }
+
+    init() {
+        let direction = JSON.parse(localStorage.getItem('direction'));
+        if (direction === null) {
+            localStorage.setItem('direction', true);
+            this.direction = true;
+        } else {
+            this.direction = direction;
+        }
```

Comment:
it doesn't really make sense to create init method if you use it just in constructor.